juniper_codegen/src/impl_object.rs

```diff
@@ -224,3 +242,47 @@ pub fn build_object(args: TokenStream, body: TokenStream, is_internal: bool) ->
     let juniper_crate_name = if is_internal { "crate" } else { "juniper" };
     definition.into_tokens(juniper_crate_name).into()
 }
+
+fn parse_argument_attrs(pat: &syn::PatType) -> Option<util::FieldAttributeArgument> {
+    let graphql_attrs = pat
+        .attrs
+        .iter()
+        .filter(|attr| {
+            let name = attr.path.get_ident().map(|i| i.to_string());
+            name == Some("graphql".to_string())
+        })
+        .collect::<Vec<_>>();
+
+    let graphql_attr = match graphql_attrs.len() {
+        0 => return None,
+        1 => &graphql_attrs[0],
+        _ => {
+            let last_attr = graphql_attrs.last().unwrap();
+            abort!(
+                last_attr.span(),
+                "You cannot have multiple #[graphql] attributes on the same arg"
+            );
```

Comment:
Not sure if we want to consider this an error or not.

Reply:
Error seems good to me